runtime/src/accounts_index.rs

```diff
@@ -33,8 +33,12 @@ use thiserror::Error;
 
 pub const ITER_BATCH_SIZE: usize = 1000;
 pub const BINS_DEFAULT: usize = 16;
-pub const BINS_FOR_TESTING: usize = BINS_DEFAULT;
-pub const BINS_FOR_BENCHMARKS: usize = BINS_DEFAULT;
+pub const BINS_FOR_TESTING: AccountsIndexConfig = AccountsIndexConfig {
+    bins: Some(BINS_DEFAULT),
+};
+pub const BINS_FOR_BENCHMARKS: AccountsIndexConfig = AccountsIndexConfig {
+    bins: Some(BINS_DEFAULT),
+};
```

Comment:
If you anticipate adding more fields, maybe it makes sense to rename these constants to indicate the type is now `AccountsIndexConfig`. Maybe like `ACCOUNTS_INDEX_CONFIG_FOR_TESTING`/etc?

I'll leave it up to you if you'd like to address that in this PR or not.